src/OXRS_LCD.cpp

```diff
@@ -1004,8 +1039,9 @@ int OXRS_LCD::_getPortLayoutGroup(int port_layout)
 */
 void OXRS_LCD::_update_input(uint8_t type, uint8_t index, int state)
 {
-  // OFF, ON, NA
-  uint16_t color_map[3] = {tft.color565(130,130,130), TFT_YELLOW, tft.color565(60,60,60)};
+  // OFF, ON, NA, DISABLED
+  uint16_t color_map[4] = {TFT_DARKGREY, TFT_YELLOW, tft.color565(60,60,60), TFT_BLACK};
```

Comment:
Can you check this - `TFT_DARKGREY` is 128,128,128 so close enough to 130,130,130 and is then consistent with the other code.